src/main/java/core/basesyntax/FieldXY.java

```diff
@@ -13,7 +13,36 @@
 public class FieldXY {
 
     public void moveRobot(Robot robot, int toX, int toY) {
-        //your code
+        if (robot.getX() < toX) {
```

Comment:
Separate this blocks to smaller methods and have a think, how to use them properly. It's a good practice to have some small methods instead of single and giant :)